packages/command/src/sfdxFlags.ts

```diff
@@ -91,7 +104,7 @@ export namespace flags {
   export type Minutes = Option<Duration> & Bounds<Duration | number>;
   export type Number = Option<number> & NumericBounds;
   export type NumericBounds = Bounds<number>;
-  export type Option<T> = Partial<IOptionFlag<Optional<T>>> & SfdxProperties & Validatable;
```

Comment:
I don't think this `Optional<T>` made sense here. The result of building an option flag shouldn't result in all properties being optional on the resulting flag. This change fixed downstream uses, I think it's correct, but another pair of eyes would be most appreciated.

Reply:
Seems fine but I'm not sure the downstream impact of it.